CityOfDetroit/bloom
# 1286/mobile filter buttons

This PR addresses issue #1286.

When using Chrome and Safari on an iOS mobile device, the "submit" and "clear" buttons are hidden on the listing filtering drawer. Typically, scrolling down hides the search toolbar but since the drawer's scrolling is not synced with the body of the html it doesn't hide. I addressed this by creating changing the height to 100% to dynamically capture the inner height whether the search toolbar is visible or not when filtering on mobile.

## How Can This Be Tested/Reviewed?

This can be tested by using an iOS device and going to the public site via google chrome or safari. From here, click on 'See Rentals, click Filter, and navigate to the bottom of the form.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
